app/javascript/components/editor-config/index.js

```diff
@@ -1,17 +1,15 @@
-// An example setup, adapted from prosemirror-example-setup
-
-import { schema, titleSchema } from './schema'
-import setupPlugins from './plugins'
+import { schema } from './schema'
+import { setupPlugins, titleSetupPlugins } from './plugins'
 
 export const options = {
-  setupPlugins,
+  setupPlugins: setupPlugins,
   schema,
   // comments: { comments: [] },
 }
 
 export const titleOptions = {
-  setupPlugins,
-  titleSchema,
+  setupPlugins: titleSetupPlugins,
```

Comment:
e.g. this could be 

```javascript
export const bodyOptions = { setupPlugins(body), schema }
export const titleOptions = { setupPlugins(title), schema }
```

Reply:
I don't understand the suggestion. How do you get setupPlugins to take in a `body` or `title` variable without also keeping the `getView` variable?